<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1485</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1451</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ToolLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>VALUES SELECTOR</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>321</width>
          <height>654</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="ValuesSelectorVerticalLayout">
         <item>
          <widget class="QLabel" name="geometryTilteLabel">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Detector features&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="detectorLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <item row="4" column="0">
            <widget class="QLabel" name="converterLabel">
             <property name="text">
              <string>Converter</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="converterComboBox"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="NbladesLabel">
             <property name="text">
              <string>Number of blades</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="bladeSpinBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="geometricalLabel">
             <property name="text">
              <string>Geometrical arrangement</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QRadioButton" name="geometricalARadioButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Single Coated</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QRadioButton" name="geometricalBRadioButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Double coated</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="thicknessBLabel">
             <property name="text">
              <string>Thickness of B4C  (μm)</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QDoubleSpinBox" name="BSpinBox">
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="lambdaLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>λ (Å)                        </string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QDoubleSpinBox" name="lambdaSpinBox">
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.800000000000000</double>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="algleLabel">
             <property name="text">
              <string>Incident angle θ (°)</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QSpinBox" name="angleSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>90</number>
             </property>
             <property name="value">
              <number>90</number>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="thresholdLabel">
             <property name="text">
              <string>E threshold (keV)</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QSpinBox" name="thresholdSpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>779</number>
             </property>
             <property name="value">
              <number>200</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Plot features&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y axis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="yAxisComboBox">
           <item>
            <property name="text">
             <string>Efficiency</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X axis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="xAxisComboBox">
           <item>
            <property name="text">
             <string>Thickness</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wavelength</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Variable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="varComboBox">
           <item>
            <property name="text">
             <string>Number of blades</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="plotButton">
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="valueSelectorLine">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="resultGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Result Viewer</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>1061</width>
          <height>921</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QWidget" name="plotWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>1001</width>
              <height>461</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="plotLayout">
             <item>
              <layout class="QVBoxLayout" name="plotLayout_2"/>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="plotTitleLAbel">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>481</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QTableWidget" name="plotTableWidget">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>510</y>
              <width>851</width>
              <height>171</height>
             </rect>
            </property>
            <column>
             <property name="text">
              <string>Color</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Thickness</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>N blades</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wavelength</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Angle</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Threshold</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Efficiency</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Max efficiency</string>
             </property>
            </column>
           </widget>
           <widget class="QPushButton" name="clearButton">
            <property name="geometry">
             <rect>
              <x>900</x>
              <y>560</y>
              <width>131</width>
              <height>81</height>
             </rect>
            </property>
            <property name="text">
             <string>Clear Plots</string>
            </property>
           </widget>
           <zorder>verticalLayoutWidget_3</zorder>
           <zorder>verticalLayoutWidget_4</zorder>
           <zorder>plotTitleLAbel</zorder>
           <zorder>plotTableWidget</zorder>
           <zorder>clearButton</zorder>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1485</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
